test/integration/action_execution.go

```diff
@@ -988,6 +993,38 @@ func newResizeActionExecutionDTO(targetSE *proto.EntityDTO, changeType int, podS
 		updatePodSpec(desiredPodSpec, 0, "requests", "memory", RESIZE_DOWN, memoryDecrement)
 		updatePodSpec(desiredPodSpec, 1, "limits", "cpu", RESIZE_DOWN, cpuDecrement)
 		updatePodSpec(desiredPodSpec, 1, "limits", "memory", RESIZE_UP, memoryIncrement)
+	case ORM_RESIZE:
+		currentSE := newContainerEntity(containerName)
```

Comment:
Maybe I am missing something, all the code below is the same for all Cases here, why are we repeating the code in each case? 
currentSE := newContainerEntity(containerName)
		containerIdx, err := findContainerIdxInPodSpecByName(podSpec, containerName)
		framework.ExpectNoError(err, "Can't find the container<%v> in the pod's spec", containerName)

		// Build the action item on cpu
		oldCpuCap := podSpec.Containers[0].Resources.Requests.Cpu().MilliValue()
		oldCpuCap := podSpec.Containers[containerIdx].Resources.Requests.Cpu().MilliValue()
		newCpuCap := oldCpuCap + cpuIncrement
		aiOnCpu := newActionItemDTO(actionType, proto.CommodityDTO_VCPU_REQUEST, oldCpuCap, newCpuCap, currentSE, targetSE)

		// Build the action item on memory
		oldMemCap := podSpec.Containers[0].Resources.Requests.Memory().Value() / 1024
		oldMemCap := podSpec.Containers[containerIdx].Resources.Requests.Memory().Value() / 1024
		newMemCap := oldMemCap + memoryIncrement*1024

Reply:
Originally, I was planning to generate the different sizes of action with different combination for different test scenarios, some have 2 action items, and some may have 4 action items.  On a second thought, that might not make sense, as all of them still go to the same code branch. Now, I change it to a unified one.  For each container,  a resize action is composed of  the 4 action items, it covers resize up and resize down, also covers request and limits commodity.